README.md

```diff
@@ -44,6 +44,82 @@ docker build -t pca-assessment .
 We welcome contributions!  Please see [here](CONTRIBUTING.md) for
 details.
 
+## Assessment JSON Field Dictionary ##
+
+The following items are included in the assessment JSON as produced by the `pca-wizard`.
+An example assessment JSON can be found [here](src/assessment/sample_assessment.json).
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| id | RV number. | string | | yes |
+| timezone | Timezone name based on [pytz](http://pytz.sourceforge.net/) timezones. | string | | yes |
+| domain | Assessment domain for GoPhish public interface. | string | | yes |
+| target_domain | Approved target domains where all email recipients must reside. | list(string) | | yes |
+| start_date| Assessment start date in 24-hr ISO format with offset. | string | | yes |
+| end_date | Assessment end date in 24-hr ISO format with offset. | string | | yes |
+| reschedule | Indicates if the assessment json is a rescheduled assessment. | boolean | | yes |
+| start_campaign | The campaign that the assessment should start at. | integer | `1` | yes |
+| groups | Consolidated list of email recipients grouped to receive campaigns, [example](#group-dictionary).| list(dictionaries)  | | yes |
+| pages |  GoPhish landing pages, [example](#page-dictionary).| list(dictionaries)  | | yes |
+| campaigns | Assessment campaigns, [example](#campaign-dictionary). | list(dictionaries) | | yes |
+
+### Group Dictionary ###
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| name | Group name in the format of `{RV number}-G{Number}`. | string | | yes |
+| targets | List of email recipients, [example](#target-dictionary). | list(dictionaries) | | yes |
+
+### Target Dictionary ###
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| first_name | Recipient's first name. | string | | yes |
+| last_name | Recipient's last name. | string | | yes |
+| email | Recipient's email address. | string | | yes |
+| position | Position name for use in creating sub-groups of recipients within the organization such as HR, IT, etc. | string | | no |
+
+### Page Dictionary ###
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| name | Page name in the format of `{RV Number}-{Page Number}-{descriptor}`. | string | | yes |
+| capture_credentials | Indicates to GoPhish if the page will forward after an action. | boolean | | yes |
```

Comment:
Is this field name and description correct?  They don't seem to match one another.

Reply:
These descriptions were focused on PCA specific use. I have changed the description to better match GoPhish [documentation](https://docs.getgophish.com/user-guide/documentation/landing-pages#capturing-credentials).

Reply:
Much nicer now! 👍